src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -253,7 +253,7 @@ CHIP_ERROR BLEManagerImpl::_Init()
     sl_rs_ble_init_sem = osSemaphoreNew(1, 0, NULL);
     sl_ble_event_sem   = osSemaphoreNew(1, 0, NULL);
 
-    wfx_rsi.ble_task = xTaskCreateStatic((TaskFunction_t) sl_ble_event_handling_task, "rsi_ble", WFX_RSI_TASK_SZ, NULL, 1,
+    wfx_rsi.ble_task = xTaskCreateStatic((TaskFunction_t) sl_ble_event_handling_task, "rsi_ble", WFX_RSI_TASK_SZ, NULL, 2,
```

Comment:
This change of task priority may not play nice with SDK users who have previously built code, or your own task. All task priorities in any `xTaskCreateStatic` call should always come from a header that allows task overriding and explains the exact priority of each task (bonus if the constants for names of the tasks are nearby).